### ISSUE TITLE
D-Bar Algorithm Plans

### ISSUE THREAD

**Issue by JeanRintoul** [liubenyuan/pyEIT#8]
Do you have any plans to integrate a D-Bar algorithm implementation into pyEIT? I'd love to use it if it were. 

https://epubs.siam.org/doi/abs/10.1137/060656930

**Comment by liubenyuan** [comment]
Yes, I do have a plan. Though d-bar is relatively hard to NOSER.

**Comment by JeanRintoul** [comment]
Great. I'm going to submit a poster proposal to an upcoming conference IEEE Brain around my work on OpenEIT. Would you mind if I put your name in acknowledgements?

**Comment by liubenyuan** [comment]
Thank you! It is my pleasure. My name can be written as Benyuan Liu, from The Fourth Military Medical University, P. R. China.

**Comment by liubenyuan** [comment]
There is a dbar implementation here: https://github.com/NablaIP/pydbar

I also have a dbar-texp branch, it is purely coded follow the tutorials on https://blog.fips.fi/tomography/eit/the-d-bar-method-for-electrical-impedance-tomography-experimental-data/,
Though, I could not quite follow the math behind dbar.

**Comment by JeanRintoul** [comment]
Thanks for the heads up.
